governance/assessments/projects/it-toto/2024-10-31.md

```diff
@@ -0,0 +1,118 @@
+# Governance Review Template
+
+What follows is a governance review and assessment for the in-toto project. This review is carried out by members of the Governance Working Group of TAG Contributor Strategy. The review may have been done because of a change in maturity level for the project, at the request of the TOC, or as a request by the project itself. If requested by the project, the review will be provided to the project maintainers. Otherwise, the review will be submitted to the TOC for their follow-up.
+
+Governance reviews contribute to the health and sustainability of the CNCF projects. By providing guidance on effective governance practices, TAG Contributor Strategy aims to ensure that projects operate efficiently, encourage diverse participation, and uphold the values of the CNCF. The governance review process is designed to be constructive and supportive, aiming to assist projects in refining their governance models and addressing any challenges they may face.
+
+Projects may ask TAG Contributor Strategy for assistance in resolving any issues uncovered by the review. The TAG is available via our [Slack channel](https://cloud-native.slack.com/archives/CT6CWS1JN), [email](https://lists.cncf.io/g/cncf-tag-contributor-strategy), [GitHub](https://github.com/cncf/tag-contributor-strategy), or by joining our weekly meetings (listed on the [CNCF public calendar](https://www.cncf.io/calendar/)).
+
+## Summary and Assessment
+
+Status: Mostly Satisfactory
+
+in-toto has evolved from a specification with a few implementations to also including other sub-projects. During this evolution, the project also evolved to a steering committee style governance which is still in its first term. Individual sub-projects are driven by maintainers.
+
+### Executing the Assessment
+
+This assessment was conducted by Bill Mulligan (Isovalent) in October 2024 based on this [commit](https://github.com/in-toto/community/tree/69d51d13ac9bb28d9b58ce263a276d73d698c98d).
+
+### Critical Items
+
+The following issues have been identified that need to be resolved before [project milestone or other requirement]:
+
+**Sub-projects**
+Multiple subprojects are not listed on the [README](https://github.com/in-toto/community/blob/main/README.md#subprojects) in the community repo including archivista and witness.
```

Comment:
@xmulligan @linsun - i dont believe we require the sub-projects to be listed in the README, rather just that they are listed. 
https://github.com/cncf/toc/blob/18f6000f66f7595ba9e169876d5b227b41f7e9b1/.github/ISSUE_TEMPLATE/template-graduation-application.md?plain=1#L128 

I also can't seem to find them in the repo, has this been resolved?

Reply:
If not in readme, we still expect it to be listed somewhere, correct? 
@SantiagoTorres mentioned to me that he will try addressing this today.

Reply:
Yea, I would expect them to be listed somewhere and the community repo seems like a good place though obviously not the only option.